### PR TITLE
added command for EZ development

### PR DESCRIPTION
**Pull request by elimelt**
Author: Elijah

## What changes were made?

Dynamic api url. `npm run prod` to connect to prod, and `npm run dev` to connect to local

## Why are these changes important/necessary?

Shut up

### CODE REVIEW

File: src/constants.ts [hunk 1]
```diff
@@ -1,3 +1,11 @@
...
 export const REFRESH_TOKEN = 'refresh';
 export const ONBOARDED = 'onboarded';
+
+const isProd = import.meta.env.PROD;
+
+export const SERVER_URL = isProd
+  ? 'https://api.swecc.org'
+  : 'http://localhost:8000';
+
+console.log('isProd:', isProd);
```
Comment (ShawnCollinge) [inline src/constants.ts:11]: Do you need this console.log?

Comment (elimelt) [reply src/constants.ts]: nope

File: src/constants.ts [hunk 2]
```diff
@@ -1,3 +1,10 @@
 export const ACCESS_TOKEN = 'access';
```
Comment (ShawnCollinge) [inline src/constants.ts:1]: I don't think we need these anymore, I know it's not part of this PR but might wanna look into it.

**Comment by ShawnCollinge** [comment]
Can you update the README as well?

**Comment by ShawnCollinge** [comment]
Per our discussion on discord because of CORS `npm run prod` won't actually work locally. I think we should still make that update with the url though to check if prod or not for connecting to the backend. If you want to just remove the npm run prod command but leave the rest that would be awesome (also revert the README).

File: src/constants.ts [hunk 3]
```diff
@@ -1,3 +1,10 @@
...
 export const REFRESH_TOKEN = 'refresh';
 export const ONBOARDED = 'onboarded';
+
+const isProd = import.meta.env.PROD;
+
+export const SERVER_URL = isProd
```
Comment (ShawnCollinge) [inline src/constants.ts:7]: Should keep this logic, make sure PROD is in the env file.

**Comment by ShawnCollinge** [comment]
Change was made in most recent PR

**Pull request closed without merging**